src/common/user-store/preferences-helpers.ts

```diff
@@ -58,10 +56,10 @@ const shell: PreferenceDescription<string | undefined> = {
 
 const colorTheme: PreferenceDescription<string> = {
   fromStore(val) {
-    return val || defaultThemeName;
+    return val || ThemeStore.defaultTheme;
```

Comment:
Can the `defaultTheme` be moved to vars (or somewhere appropriate) and imported from there?

Reply:
Yup